File: Examples/test-suite/php/sym_runme.php
<?php

require "tests.php";

// No new functions
check::functions(array());
// New classes
check::classes(array('Flim','Flam'));
// No new vars
check::globals(array());

$flim=new flim();
$flam=new flam();

check::equal($flim->hulahoops(),"flim-jam","flim()->hulahoops==flim-jam");
check::equal($flim->jar(),"flim-jar","flim()->jar==flim-jar");
check::equal($flam->jam(),"flam-jam","flam()->jam==flam-jam");
check::equal($flam->jar(),"flam-jar","flam()->jar==flam-jar");

check::done();

File: Examples/test-suite/lua/sym_runme.lua
require("sym")

local flim=sym.Flim()
local flam=sym.Flam()
assert(flim:hulahoops() == "flim-jam","flim:hulahoops()")
assert(flim:Jar() == "flim-jar","flim:Jar()")
assert(flam:Jam() == "flam-jam","flam:Jam()")
assert(flam:Jar() == "flam-jar","flam:Jar()")
